src/rest/json.cpp

```diff
@@ -482,6 +577,224 @@ std::string Error2JsonString(HttpStatusCode aErrorCode, std::string aErrorMessag
     return ret;
 }
 
+std::string Dataset2JsonString(const otOperationalDataset &aDataset)
+{
+    cJSON      *node = cJSON_CreateObject();
+    std::string ret;
+
+    if (aDataset.mComponents.mIsActiveTimestampPresent)
+    {
+        cJSON_AddItemToObject(node, "ActiveTimestamp", cJSON_CreateNumber(aDataset.mActiveTimestamp.mSeconds));
```

Comment:
I went for seconds only for the sake of simplicity. My thinking was that we can still extend the code to accept a JSON object as `ActiveTimestamp` if needed later. It is pretty easy to check which datatype is passed to `ActiveTimestamp`.

That said, it would break the API if we change the format of the JSON structure we return, so maybe a separate field is better still.

That said, I am also ok to extend the API to take the full timestamp as JSON object. I guess the JSON would end up looking like this:


```json
"ActiveTimestamp": {
    "Seconds": 123,
    "Ticks": 123,
    "Authoritative": false
}
```

Reply:
I've added such an implementation.

Reply:
Thanks!